Test/TMU_test/TMU_test.c

```diff
@@ -58,173 +61,287 @@ void stub_comsumer_fun(void)
 	
 }
 
+void checkTestCase(sint16_t s16_expected_return , sint16_t s16_fun_return )
+{
+	static uint8_t u8_testCaseNumber = 0;
+	printf("testCase %3d:	expected_return : %3d	 fun_return: %3d	",u8_testCaseNumber,s16_expected_return,s16_fun_return);
+	/*check on test case*/
+	if (s16_fun_return == s16_expected_return)
+	printf("Passed.\n");
+	else
+	printf("Failed.\n");
```

Comment:
It would be better to print the actual an the expected results for easy tracking and modifications.